elliottlib/errata_async.py

```diff
@@ -34,9 +34,9 @@ async def login(self):
     async def _make_request(self, method: str, path: str, parse_json: bool = True, **kwargs) -> Union[Dict, bytes]:
         if "headers" not in kwargs:
             kwargs["headers"] = self._headers
-        async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(limit=32, force_close=True)) as session:
+        await self.login()
```

Comment:
how does this work? wouldn't it be redundant to attempt to login with every request?

Reply:
the login method is to make sure the SecurityContext is set for the request
and actually in this version only cve_package_exclusion is using this, only four requests will be called